TrafficCapture/dockerSolution/build.gradle

```diff
@@ -16,8 +16,6 @@ clean.doFirst {
 }
 
 dependencies {
-    implementation project(':TrafficCapture:trafficCaptureProxyServer')
-    implementation project(':TrafficCapture:trafficReplayer')
```

Comment:
I assume these dependencies are still being added somehow, but I don't see where. Ah, wait, actually, these are no longer dependencies because we don't need them (meaning the kafka printer) on the console?

Reply:
I don't know what the dependencies were for - I put them in a while ago when I was in a state of exasperation and exhaustion.  We wire up dependencies for the images separately & it seems that that should be enough.